translations/translation_utility.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+# You may add your name if you have contributed to this file.
+echo -ne "
+    ╔══════════════════════════════════════╗
+    ║                                      ║
+    ║ Linux/MacOS Translation utility 1.0  ║
+    ║     Powered by David Copperfield     ║
+    ║           A MuseScore User           ║
+    ║                                      ║
+    ╚══════════════════════════════════════╝\n\n"
+# Functions
+function Countdown() { # Countdown timer, format: Countdown [time in sec]
+    _countdown=$1
+    while [[ $_countdown -gt 0 ]]; do
+        echo -ne "The programme will exit in $_countdown sec."
+        ((_countdown--))
+        sleep 1 #time interval
+        echo -ne "\r"
+    done
+}
+function Confirmation() { # Process Yes/No/Quit response
+    while :; do
+        echo -e "Confirm? (y/n) "
+        read
+        case ${REPLY} in
+        [Yy]* | [Oo][Kk] | 1) # Approved
+            return 3
+            ;;
+        [Qq]uit* | [Ee]sc* | [Ee]xit | -1) # Leave
+            Countdown 5
+            exit
+            ;;
+        "") # Avoid mis-press the Enter
+            continue
+            ;;
+        *) # Not confirmed
+            return
+            ;;
+        esac
+    done
+}
+# Functions End
+
+# ===== Modules ====
+Check_bin() {                             #Check if the bin path exists
+    bin_path=$HOME/Qt/5.15.2/clang_64/bin # Default bin path
+
+    until [[ -d $bin_path ]]; do # Find the default bin path
+        echo =========
+        echo -e "\033[31mPath to lupdate not found.\033[0m"
+        echo -e "Please provide a path to /Qt/(version)/clang_64/bin."
```

Comment:
This is Mac specifig (missed this last time) maybe make it more general
`Please provide a path to the location of the lupdate/lrelease programs`

Reply:
Also in that loop make sure that lupdate/lrelease are really found, like using `which lupdate`?

Reply:
I think "which" will be better. ￼

Reply:
Maybe I will add a system check. 
I wonder what could be the default lupdate / lrelease path in Linux￼￼￼

Reply:
Doesn't really matter, it may or may not be in PATH in any case. Check whether it is and ask for the part to is if not. Shpould work in Mac __and__ Linux